openerp/tools/translate.py

```diff
@@ -843,16 +843,20 @@ def push_local_constraints(module, model, cons_type='sql_constraints'):
         lambda m: m['name'],
         registry['ir.module.module'].search_read(cr, uid, [('state', '=', 'installed')], fields=['name']))
 
-    path_list = list(openerp.modules.module.ad_paths)
+    path_list = [(path, True) for path in openerp.modules.module.ad_paths]
     # Also scan these non-addon paths
     for bin_path in ['osv', 'report' ]:
```

Comment:
Maybe `['osv', 'report', 'service', 'modules', 'tools', ]` will be better here to really collect all terms to translate